### PR TITLE
allows to define another osm map in config.js

### PR DESCRIPTION
**Pull request by khassel**
e.g. a map without colors

grafik

### CODE REVIEW

File: MMM-RAIN-MAP.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ Module.register("MMM-RAIN-MAP", {
 		markers: [],
 		onlyOnRain: false,
 		opacity: 0.65,
+		osmmapurl: "https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",
```
Comment (jalibu) [inline MMM-RAIN-MAP.js:23]: please rename to osmMapUrl

File: README.md [hunk 2]
```diff
@@ -120,6 +120,12 @@ git clone https://github.com/jalibu/MMM-RAIN-MAP.git
...
 | `mapTypeId`        | The map type to display.<br>Possible values: `'roadmap', 'satellite', 'hybrid', 'terrain'`. <br><br>**Type:** `string` <br> **Default value:** `'terrain'`                            |
 
+### OSM Maps only options
+
+| Option             | Description                                                                                                                                                                           |
+| ------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `osmmapurl`        | Url of the used map. You can find alternative maps [here](https://wiki.openstreetmap.org/wiki/Tile_servers) in the column `tiles url`. <br> Before inserting the url into `config.js`you must delete the `$` signs before the {x},{y},{z}. <br>E.g.instead of using `https://tiles.wmflabs.org/bw-mapnik/${z}/${x}/${y}.png` <br> it has to be changed to `https://tiles.wmflabs.org/bw-mapnik/{z}/{x}/{y}.png` |
```
Comment (jalibu) [inline README.md:127]: osmMapUrl

Comment (jalibu) [reply README.md:127]: Can't we just add a line of code in utils.js to automate that $ replacement?

File: utils.js [hunk 3]
```diff
@@ -22,7 +22,7 @@ class Utils {
 				zoomControl: false,
 				attributionControl: false,
 			}).setView([module.config.lat, module.config.lng], module.config.zoom);
-			L.tileLayer("https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png").addTo(
+			L.tileLayer(module.config.osmmapurl).addTo(
```
Comment (jalibu) [inline utils.js:25]: L.tileLayer(module.config.osmmapurl.split('$').join('')).addTo(

**Comment by khassel** [comment]
pushed a new commit this morning, ready to merge.

**Pull request merged**